UDV-Core/src/Modules/ConsultDoc/DocumentBrowser.js

```diff
@@ -26,10 +26,12 @@ export function DocumentBrowser(browserContainer, documentController) {
   this.isOrientingDoc = false;
   this.isFadingDoc = false;
   this.fadeAlpha = 0;
-  this.docIndex = 1;
+  this.docIndex=1;
   this.isStart = true; //dirty variable to test if we are in start mode
   this.currentMetadata;
 
+  this.numberDocs;
```

Comment:
Do you really need this attribute since sometimes you use this.documentController.setOfDocuments.length instead ?

Reply:
Not necessarily, but it is good for clarity. I changed the "this.documentController.setOfDocuments.length" to "this.numberDocs"